### ISSUE TITLE
Support formsets

### ISSUE THREAD

**Issue by bedros** [barseghyanartur/django-fobi#49]
repeated item in a form similar to the image in the link below

this type of form is very common in service business, such as repair shop,
What I'm thinking of is like entering (Qty, Material, price, amount) then ask user (or checkbox) to enter more of the same, once user enters all items,  press done with items, then go to fill the rest of the form

this enhancement would be really nice, since it's very common in service business

[paper form](https://www.printez.com/image/cache/Wide-Carbonless-Work-Orders-BussinessForm2013/213-large.jpg)

**Comment by barseghyanartur** [comment]
@bedros:

This feature is already for a long time on the TODOs list. I don't think it will land here before django-rest-integration part and react-frontend for the django-rest-framework integration are ready. Thus, not in a nearest future, but I'm adding it to the unscheduled of TODOs milestone.

**Comment by bedros** [comment]
great thank you; 

can you guys do vue.js or similar instead of react; I'm not a fan of facebook patent clause in react license. or if react front end is optional, then that works too.

Thanks for the great project, the demo site is a big plus as well.

**Comment by barseghyanartur** [comment]
@bedros:

Sure, it will be a stand-alone react component which will not be directly included in this package. Regarding the license, it might change in future to BSD3  (but that's very unsure yet), which only allows more.

**Comment by bedros** [comment]
you mean pure BSD3, it's already BSD3 but with extra stuff on top

[react license ](http://react-etc.net/entry/your-license-to-use-react-js-can-be-revoked-if-you-compete-with-facebook)

**Comment by barseghyanartur** [comment]
@bedros:

No, I mean the license of `django-fobi` and the upcomng react integration app, not the `react` itself.

**Comment by bedros** [comment]
Hi @barseghyanartur 

I looked into how to do the formset using existing features, so I used the wizard and tried to repeatedly ask user to fill same form  5 times (Qty, Description, price, etc) for each time,  however ,the wizard will not let me use same form multiple times, so I ended up recreating the same form under different names (sevices0, services1, etc) and then I got into another problem, when the wizard moves from one form page to another, there's no description on top of the form page telling the user what to do with the current form page. (in my case, "leave form empty and click submit  if no more services') 

So, basically, it'll be a simple change to allow the wizard to reuse the form multiple times instead of filtering out already used forms from forms dropdown list.

And, it would be nice to have a simple label to display on top page of each form page, telling the user what to do with this form page.

Thank you so much.

**Comment by barseghyanartur** [comment]
@bedros:

Wizard functionality is very new and I didn't have time to "polish" the UI, but navigation, form titles and other improvements (in short - a perfect UI) is on the TODO list as well (to be implemented shortly).

**Comment by barseghyanartur** [comment]
@bedros:

FYI, wizards GUI has been improved. Form has got a new field called "title" which is displayed in wizard steps. Navigation has been also added. Upgrade to 0.9.16

Please, separate your issues. Off-topics make things difficult.

Regarding reusing of forms multiple times in a single wizard - you may make as many forms as you like and link them in one wizard. If your forms are complex, you can export them and import again.

Regarding the label on top of the form page - you can do that in the form itself (use content plugin).

**Comment by bedros** [comment]
great thank you, I'll try it out.
